### PR TITLE
tls-assets: add kube-apiserver-to-kubelet CA to kubelet client CAs

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

**Comment by sttts** [comment]
/assign @abhinavdahiya @sjenning 

Is this necessary to make apiserver->kubelet communication (log, exec) work with the bootstrap control plane?

Compare https://github.com/openshift/cluster-kube-apiserver-operator/pull/355/files

File: pkg/asset/tls/kubelet.go [hunk 1]
```diff
@@ -48,6 +48,7 @@ var _ asset.Asset = (*KubeletClientCABundle)(nil)
 func (a *KubeletClientCABundle) Dependencies() []asset.Asset {
 	return []asset.Asset{
 		&KubeletCSRSignerCertKey{},
+		&KubeAPIServerToKubeletCABundle{},
```
Comment (deads2k) [inline pkg/asset/tls/kubelet.go]: This looks correct.  This is the bundle that can be used to verify the kube-apiserver's client cert to the kubelet.

**Comment by abhinavdahiya** [comment]
Please update the commit and PR message. Also can you provide more details on why this is required? Was it working before? If it wasn't how this fixes it? And could we have a test (origin e2e) that can validate we do not regress in the future?

**Comment by abhinavdahiya** [comment]
also I would like to see the Certificate doc updated and linked..

**Comment by sttts** [comment]
@abhinavdahiya was hoping you or @deads2k can tell me why it still does not work, with this change and the other one in the kas-operator.

**Comment by abhinavdahiya** [comment]
It looks like we don't need this. Please feel free to reopen, if this change is still required.

/close

**Pull request closed without merging**